clio-server/src/test/scala/org/broadinstitute/clio/server/webservice/ReadGroupWebServiceSpec.scala

```diff
@@ -20,7 +20,7 @@ class ReadGroupWebServiceSpec
   it should "postMetadata with OnPrem location" in {
     val webService = new MockReadGroupWebService()
     Post(
-      "/metadata/v1/barcodeOnPrem/3/libraryOnPrem/OnPrem",
+      "/metadata/barcodeOnPrem/3/libraryOnPrem/OnPrem",
```

Comment:
The `v1` path segment got pulled out of `webService.postMetadata` and into `webService.readGroupRoutes`. Should we use the read group routes value in all of these tests instead of the individual sub-routes?